python/telemetry/snet.py

```diff
@@ -168,10 +100,91 @@ def _decode(self, obj, context, path=None):
     'TargetData_ControlIsActive' / Int8ul  # flag, really
     )
 
+LTUFrameHeader = BitStruct(
+    'SrcId' / BitsInteger(7),
+    'DstId' / BitsInteger(7),
+    'FrCntTx' / BitsInteger(4),
+    'FrCntRx' / BitsInteger(4),
+    'SNR' / BitsInteger(4),
+    'AiTypeSrc' / BitsInteger(4),
+    'AiTypeDst' / BitsInteger(4),
+    'DfcId' / BitsInteger(2),
+    'Caller' / Flag,
+    'Arq' / Flag,
+    'PduTypeId' / Flag,
+    'BchRq' / Flag,
+    'Hailing' / Flag,
+    'UdFl1' / Flag,
+    'PduLength' / BitsInteger(10),
+    'CRC13' / BitsInteger(13),
+    'CRC5' / BitsInteger(5),
+    Padding(2)
+    )
```

Comment:
Please do not needlessly move the location of this (and other elements) in the source code file. If it's move, it's difficult to tell if there are any changes by looking at the diff.

Reply:
you are right with this one, it was really not necessary to be moved, however, `PDUFrameHeader` originally `SNETFrameHeader` had to be moved because the introduction of the `Switch()` for `UserData` and its references to `ADCSTelemetry` and `EPSTelemetry`

Reply:
moved it back